src/shared/components/query/CancerStudySelector.tsx

```diff
@@ -330,18 +342,28 @@ export default class CancerStudySelector extends React.Component<ICancerStudySel
 
                         <div className="checkbox" style={{marginLeft: 19}}>
                             <If condition={shownStudies.length > 0}>
-                                <label>
-                                    <input type="checkbox"
-                                           data-test="selectAllStudies"
-                                           style={{top: -2}}
-                                           onClick={this.handlers.onCheckAllFiltered}
-                                           checked={shownAndSelectedStudies.length === shownStudies.length}
-                                    />
-                                    <strong>{(shownAndSelectedStudies.length === shownStudies.length) ?
-                                        `Deselect all listed studies ${(shownStudies.length < this.store.cancerStudies.result.length) ? "matching filter" : ""} (${shownStudies.length})` :
-                                        `Select all listed studies ${(shownStudies.length < this.store.cancerStudies.result.length) ? "matching filter" : ""}  (${shownStudies.length})`}
-                                    </strong>
-                                </label>
+                                <If condition={this.logic.mainView.showQuickSelect}>
+                                    <div className={styles.quickSelect}>
+                                        Quick select: <button onClick={this.selectTCGAPanAtlas} className={"btn btn-default btn-xs"}>TCGA Pancancer Atlas studies</button> <button className={"hidden btn btn-default btn-xs"}>All TCGA PanCan</button>

```

Comment:
whats this second button about? "All TCGA PanCan"

Reply:
nice spot.   it's hidden.  i thought there might be more buttons.  will kill it.